CHANGELOG.md

```diff
@@ -4,6 +4,22 @@ All notable changes to this project will be documented in this file, per [the Ke
 
 ## [Unreleased] - TBD
 
+## [1.3.0] - 2024-03-14
```

Comment:
Assuming this should be 05?
```suggestion
## [1.3.0] - 2024-05-14
```

Reply:
lol yep, I've yet to figure out time travel